pkg/nsx/services/subnetport/subnetport.go

```diff
@@ -130,6 +130,9 @@ func (service *SubnetPortService) CreateOrUpdateSubnetPort(obj interface{}, nsxS
 	if (*nsxSubnet).DhcpConfig != nil && *nsxSubnet.DhcpConfig.EnableDhcp {
 		enableDHCP = true
 	}
+	if nsxSubnet.SubnetDhcpConfig != nil && nsxSubnet.SubnetDhcpConfig.Mode != nil && *nsxSubnet.SubnetDhcpConfig.Mode != "DHCP_DEACTIVATED" {
```

Comment:
ditto. Use a const instead of hardcode `DHCP_DEACTIVATED` here?

Reply:
Updated, thanks!